Assets/MRTK/SDK/Experimental/ScrollingObjectCollection/ScrollingObjectCollection.cs

```diff
@@ -279,18 +278,17 @@ public float AnimationLength
             set { animationLength = value; }
         }
 
-        [Tooltip("Number of columns or rows in respect to ViewableArea and ScrollDirection.")]
+        [Tooltip("Number of items in a line on up-down scroll direction or number of items in a column on left-right scroll direction.")]
         [SerializeField]
-        [Range(1, 500)]
-        private int tiers = 2;
+        private int itemsPerTier = 1;
 
         /// <summary>
-        /// Number of columns or rows in respect to <see cref="ViewableArea"/> and <see cref="ScrollDirection"/>.
+        /// Number of items in a line on up-down scroll direction or number of items in a column on left-right scroll direction.
         /// </summary>
-        public int Tiers
+        public int ItemsPerTier
         {
-            get { return (tiers > 0) ? tiers : 1; }
-            set { tiers = value; }
+            get { return (itemsPerTier > 0) ? itemsPerTier : 1; }
```

Comment:
It seems like itemsPerTier should never be equal to or less than zero, do we want to Debug.Assert instead?